ports/sonictt16b/port.json

```diff
@@ -24,7 +24,7 @@
     "rtr": false,
     "exp": false,
     "runtime": null,
-    "reqs": [],
+    "reqs": ["!arkos"],
```

Comment:
You have a if arkos library condition to use other libs. And then you exclude the os completely?

Reply:
Yes because people can be very stubborn.